src/components/Search.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Displayresults from './Displayresults'
+class Search extends React.Component {
+constructor(){
+    super()
+    this.state = {
```

Comment:
It looks like the search query is stored both in `Search` and `App`. It would better to avoid state duplication and pass the search query down from `App` to `Search`